<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buzzer</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>460</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0">
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_01">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_02">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_03">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_04">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_05">
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_06">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_07">
        <property name="text">
         <string>G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_08">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_09">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="text">
         <string>E</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
